package.json

```diff
@@ -10,7 +10,7 @@
     "test-integration": "npm run clean && ./node_modules/.bin/mocha test/integration/test/*.js && ./node_modules/.bin/mocha test/integration/test/no_travis/*.js",
     "test-integration-travis": "npm run clean && ./node_modules/.bin/mocha test/integration/test/*.js",
     "clean": "npm run clean_db && npm run clean_picture",
-    "clean_db": "rm -f ~/.local/share/foxbox/users_db.sqlite",
+    "clean_db": "rm -f ~/.local/share/foxbox/*.sqlite",
```

Comment:
The effect will happen locally if you run `npm run clean_db` or `npm run test-integration`